src/components/ListExport/Popup/Popup.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './popup.css'
+
+export default function Popup({ active, handleToHide },props){
+  return(
+    <div className={'popup ' + (active ? 'active' : '')}>
+      <div className='popup-window'>
+      <button onClick={handleToHide} className='btn-close-modal'>
+         <Icon name="/img/x.svg" />
```

Comment:
Icon needs to be imported. and remove the `button` wrapper and just attach `onClick` to this `Icon` element

Reply:
`<Icon name="x.svg" onClick={handleToHide} />`